src/lib/mina/fetch.ts

```diff
@@ -77,6 +78,26 @@ type NetworkConfig = {
   lightnetAccountManagerEndpoint: string;
 };
 
+type ActionsQueryInputs = {
+  publicKey: string;
+  actionStates: ActionStatesStringified;
+  tokenId?: string;
+}
```

Comment:
@mitschabaude, this type is only used in the file `fetch.ts`.  The graphQL query accepts the destructured inputs.

I made this type because previously we had the function param:

```
accountInfo: {
    publicKey: string;
    actionStates: ActionStatesStringified;
    tokenId?: string;
  },
```

When I made the new function `createActionList`, I duplicated this, so I made a custom type.  I didn't think `AccountInfo` was a good type name, but I see how the one I chose is also confusing.  WDYT?  Is there a better type name, or is it better to just leave the explicit definition in 2 function definitions?

Reply:
hm ok, you can leave it here as well :D